index.html

```diff
@@ -2,9 +2,175 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <link rel="stylesheet" href="styles.css">
+    <title>Rekupe</title>
 </head>
 <body>
 
+<div class="gradient-background-color">
+    <section class="navigation">
+        <nav class="navigation-container">
+            <ul class="navigation-content">
+                <li>Case studies</li>
+                <li>Services</li>
+                <li>Contact us</li>
+            </ul>
+        </nav>
+    </section>
+    <section class="document-titel-area">
+        <article>
+            <h1 class="document-titel">Your website is an investment in the success of your business.</h1>
+            <p class="document-under-titel">We are here to help you succeed</p>
+            <button class="document-titel-button">let's work together</button>
+            <br><br>
```

Comment:
Self-closing tag gebruiken
```suggestion
            <br/><br/>
```